<?xml version="1.0" encoding="utf-8"?>

<launch>

    <!-- # gazebo -->
    <include file="$(find docking_gazebo)/launch/robot.launch" />

    <!-- # static tf -->
    <node pkg="tf" type="static_transform_publisher" name="br1" args="0 0 0 0 0 0 1 map odom 100" />

    <!-- # obstacle_detector -->
    <include file="$(find obstacle_detector)/launch/nodes.launch" />

    <!-- # move_base -->
    <include file="$(find docking_sim)/launch/_move_base.launch">
        <arg name="use_map" value="false" />
    </include>

    <!-- # docking -->
    <node pkg="docking_sim" type="docking.py" name="docking" output="screen">
        <param name="ns_move_base" value="/move_base" />
    </node>

    <!-- # rviz -->
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find docking_sim)/rviz/nav.rviz" />

</launch>
